myko/src/lib/visualisation/sketch.js

```diff
@@ -103,6 +110,14 @@ export function draw(p5) {
     atm.shows(index);
     atm.edge();
   }
+
+  weatherOn = true;
+  weatherPosition = 2;
+  weatherType = 'snow';
+  if (weatherOn && weatherType) {
+    p5.image(precipitationCloud, 500, 250);
+    makeWeather(weatherType, weatherPosition, p5);
```

Comment:
I think this could be clearer as well, weatherPosition could probably be the same for the precipitationCloud except maybe higher now that I think about it more

Reply:
Cool I was just looking at this myself! 
I don't totally understand what your vision is for the weather cloud.. Because it's hardcoded, I'm not sure where in the visualisation you want it to be.. is it somewhere on the shelf? Or something else? Or do you want it to be able to have different positions? 
Either way, to make it responsive I think it could work to use the image location arrays. So if you wanted the first cat position (the one highest up in the middle) you would say: imagePositions[0][0] for one of the shelf positions imagePositions[2][n] etc

Reply:
also, is weather supposed to happen when there are a couple of activity images there? In that case, how are you thinking about placing the cloud so it doesn't cover up activity images?